src/complex.rs

```diff
@@ -132,6 +133,38 @@ impl<T: Clone + Float> Complex<T> {
         let (r, theta) = self.to_polar();
         Complex::from_polar(&(r.sqrt()), &(theta/two))
     }
+    
+    /// Raises a number to a floating point power.
+    #[inline]
+    pub fn powf(&self, value: T) -> Complex<T> {
+        let (r, theta) = self.to_polar();
+        Complex::from_polar(&(r.powf(value)), &(theta*value))
+    }
+    
+    /// Returns the logarithm of the number with respect to an arbitrary base.
+    #[inline]
+    pub fn log(&self, value: T) -> Complex<T> {
+        let (r, theta) = self.to_polar();
+        // Comment for code review:
+        // There is for sure a faster way to get the constant, but I don't know how to do that in this generic context 
+        // Any hints?
+        let e = T::one().exp(); 
```

Comment:
`powf(self, exp: T)` sounds fine here.  Let's not bother changing existing functions though -- that's the sort of thing we could clean up if we ever need to make other changes to them.  Otherwise it's unnecessary churn.  You're right though that changing parameter names is not a breaking change.